src/main/java/com/school/schedule/app/School.java

```diff
@@ -32,6 +37,10 @@ public float getNumberOfStaff() {
 
     // Setter Methods
 
+    public void setName( String address ) {
```

Comment:
I see there is a mistake here. You need to use `name` instead of `address`

Reply:
Good catch! I've made the fix 👍